test/parser/LiteralOrIriTest.cpp

```diff
@@ -339,3 +339,26 @@ TEST(LiteralTest, removeDatatypeOrLanguageTag) {
   literal.getLiteral().removeDatatypeOrLanguageTag();
   EXPECT_THAT("Hello World!", asStringViewUnsafe(literal.getContent()));
 }
+
+// _______________________________________________________________________
+TEST(LiteralTest, replaceContent) {
+  LiteralOrIri literal = LiteralOrIri::literalWithoutQuotes(
+      "Hello!", Iri::fromIriref("<http://www.w3.org/2001/XMLSchema#string>"));
+  literal.getLiteral().replaceContent("Hello!");
```

Comment:
Replace it by something different with the same length, s.t. the test becomes more expressive.